### ISSUE TITLE
How to apply StatsModels for partial least squares regression

### ISSUE THREAD

**Issue by leavor** [statsmodels/statsmodels#5978]
How to apply StatsModels for partial least squares regression?
There is PLSRegression in sklearn, but some statistical results are missing, compared to StatsModels.GLM

**Comment by leavor** [comment]
The minitab PLS results are quite different from sklearn.PLSRegression

**Comment by bashtage** [comment]
There are probably different normalizations used.  Are the fitted values the same?

SM has no support for PLS regression since this is mostly used in ML/prediction, and so has not been a priority.

**Comment by bashtage** [comment]
If you have doubts about scikit-learn's implementation, you should open an issue on scikit's github. 

You should also see the same R2 for the same number of components.

**Comment by bashtage** [comment]
Retitling as an enhancement request.

**Comment by olivertomic** [comment]
@leavor , this comment is one year late, but if it is still relevant you may want to take a look at our python implementation of PLS (https://github.com/olivertomic/hoggorm). Our results are benchmarked against pls in R (https://www.rdocumentation.org/packages/pls/versions/2.7-2) and The Unscrambler software (https://www.camo.com/unscrambler/).
